src/pages/guidelines/index.astro

```diff
@@ -62,21 +55,24 @@ import GuidelinesRespec from "@/components/respec/GuidelinesRespec.astro";
 				<li>What are your thoughts on assertions?</li>
 				<li>Does the "Which foundational requirements apply?" section in <a href="#image-alternatives">Image Alternatives</a>, <a href="#text-appearance">Text Appearance</a>, or <a href="#keyboard-focus-appearance">Keyboard Focus Appearance</a> help you understand how to use the requirements?</li>
 			</ul>
-			<p>Additionally the Working Group would welcome any research that supports requirements or assertions.</p>
+			<p>Additionally, the Working Group welcomes any research that supports requirements or assertions.</p>
 	
-			<p>To provide feedback, please file a <a href="https://github.com/w3c/wcag3/issues">GitHub issue</a> or email <a href="mailto:public-agwg-comments@w3.org?subject=WCAG%203.0%20public%20comment">public-agwg-comments@w3.org</a> (<a href="https://lists.w3.org/Archives/Public/public-agwg-comments/">comment archive</a>).</p>
+			<p>To provide feedback, please open a new issue in the <a href="https://github.com/w3c/wcag3/issues">wcag3 GitHub repository</a>. Create a separate GitHub issue for each topic, rather than commenting on multiple topics in a single issue.</p>
```

Comment:
```suggestion
			<p>To provide feedback, please open a new issue in the <a href="https://github.com/w3c/wcag3/issues">WCAG 3 GitHub repository</a>. Create a separate GitHub issue for each topic, rather than commenting on multiple topics in a single issue.</p>
```